packages/form-core/src/validate/ValidateMixin.js

```diff
@@ -85,6 +85,8 @@ export const ValidateMixinImplementation = superclass =>
          * By default, just like the platform, only one message (with highest prio) is visible.
          */
         _visibleMessagesAmount: { attribute: false },
+
+        __childModelValueChanged: { attribute: false, type: Boolean },
```

Comment:
Ah small detail but `type: Boolean` is essentially redundant as all it does it tell lit-html how to reflect to an attribute, but we don't reflect to attribute.

Reply:
Ah, true indeed 👍